schedule/models/events.py

```diff
@@ -47,10 +47,10 @@ def get_for_object(self, content_object, distinction='', inherit=True):
 
 
 @python_2_unicode_compatible
-class Event(models.Model):
+class AbstractEvent(models.Model):
     '''
```

Comment:
I know it's not your line, but could you please replace ''' with """ in docstring?
https://www.python.org/dev/peps/pep-0008/#documentation-strings